src/products/components/ProductVariants/utils.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  AvailableColumn,
+  DatagridCell
+} from "@saleor/components/Datagrid/types";
+import { ProductDetailsVariantFragment } from "@saleor/graphql";
+import { IntlShape } from "react-intl";
+
+import messages from "./messages";
+
+const isStockColumn = /^stock:(.*)/;
+const isChannelColumn = /^channel:(.*)/;
+const isAttributeColumn = /^attribute:(.*)/;
```

Comment:
capturing groups are not necessary here:
```
/^stock:.*/
```

Reply:
Actually it is, look below - function below uses `match` to get to warehouse/channel/attribute id